src/makeCalculator.js

```diff
@@ -4,7 +4,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add: (num) => result + num,
+
+    subtract: (num) => result - num,
+
+    multiply: (num) => result * num,
+
+    divide: (num) => (num !== 0 ? result / num : NaN),
```

Comment:
The `divide` method should update the `result` variable. Consider changing it to `result /= num;` and ensure that division by zero is handled appropriately.